src/components/ActionBar/README.md

```diff
@@ -0,0 +1,117 @@
+# ActionBar
```

Comment:
I think should use `mdx` format for docs

Reply:
In the component folder we use the usual normal markdown